projects/packages/jetpack-mu-wpcom/src/features/wpcom-comments/wpcom-comments.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Adds a "Like" action to comment rows and handles the required scripts in the admin area.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Adds a "Like" action link to comment rows.
+ *
+ * @param array      $actions The existing comment row actions.
+ * @param WP_Comment $comment The comment object for the current row.
+ * @return array Modified array of comment row actions.
+ */
+function wpcom_add_comment_like_action( $actions, $comment ) {
+	$comment_id = $comment->comment_ID;
+	$blog_id    = esc_attr( get_current_blog_id() );
+	$nonce      = wp_create_nonce( 'like_comment_' . $comment_id . '_' . $blog_id );
+
+	$feedback = esc_html( comment_like_get_feedback_esc( $comment_id ) );
+
+	$actions['like'] = sprintf(
+		'<a data-nonce="%s" data-comment="%s" data-blog="%s" href="#" class="comment-like-link"><span>%s</span></a>',
+		esc_attr( $nonce ),
+		$comment_id,
+		$blog_id,
+		$feedback
```

Comment:
Escaping should be done "at the edge", so here rather than when assigning `$blog_id` or `$feedback`:

```suggestion
		esc_attr( $comment_id ),
		esc_attr( $blog_id ),
		esc_html( $feedback ),
```